src/tango/mvcc/test_mvcc.c

```diff
@@ -0,0 +1,26 @@
+#include "../../util/fd_util.h"
+#include "fd_mvcc.h"
+
+int
+main( int argc, char ** argv ) {
+  fd_boot( &argc, &argv );
+
+  fd_mvcc_t mvcc = { .version = 0 };
```

Comment:
This should use new and join so that those functions actually get tested

Reply:
And maybe testing the app region would be good too. I think the ideal test would spin up two threads and hammer it, but a single threaded test is fine